exercises/javascript-1-a/README.md

```diff
@@ -0,0 +1,34 @@
+## Introduction
+
+This is Part 1 of our research into how people write JavaScript differently. This exercise should take 10-15 minutes. There is no right or wrong way to approach it - just do what feels the most natural to you. Feel free to use books, Google or Stack Overflow, just like you would if you were programming normally, but please don't actively look for a solution to the problem as this will negatively affect the research.
+
+## Instructions
+
+You're a magician and you handle a deck of cards. In order to correctly execute your magic trick, you need to be able to _move_ a card from one position to another position. That is, you need to be able to `rearrange` the deck. Naturally, you want to be able to move in _both directions_, as well as use a negative numbers for the `from` and `to` values (so you can say "from the top" or "from the bottom" -- it's all about the magic ✨).
```

Comment:
I think move the negative numbers bits out of here, as they detract from the story/metaphor, and add them into the spec below.

```suggestion
You're a magician and you handle a deck of cards. In order to correctly execute your magic trick, you need to be able to _move_ a card from one position to another position. That is, you need to be able to `rearrange` the deck. Naturally, you want to be able to move cards in _both directions_, and be able to say "from the top of the deck" or "from the bottom of the deck".
```

Reply:
Sure!